### PR TITLE
Dryrun

### PR DESCRIPTION
**Pull request by jonathanmorley**

### CODE REVIEW

**Comment by jonathanmorley** [comment]
@brentm5

**Comment by brentm5** [comment]
It failed

File: Policyfile.rb [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 default_source :community
 
+cookbook 'no_op', github: 'livinginthepast/windows-noop-cookbook'
```
Comment (brentm5) [inline Policyfile.rb]: Why not just include a cookbook internally?

File: test_file [hunk 2]
```diff
@@ -0,0 +1 @@
+[2016-09-01T11:31:03-04:00] [1;31mERROR: No password provided. Unable to push ssh key to stash[0m
```
Comment (brentm5) [inline test_file]: Generic?

File: client.rb [hunk 3]
```diff
@@ -5,6 +5,7 @@
 local_mode                  true
 chef_zero.enabled           true
 log_level                   :error
+add_formatter               'doc'
 node_name                   (ENV['SUDO_USER'] || ENV['USER'] || ENV['USERNAME']).downcase
```
Comment (brentm5) [inline client.rb]: Default value?

File: workup.rb [hunk 4]
```diff
@@ -0,0 +1,118 @@
+require 'mixlib/shellout'
```
Comment (brentm5) [inline workup.rb]: Why not just turn this into a gem and install it so you can run `chef exec workup` ?

**Comment by brentm5** [comment]
Add a description of what this PR is suppose to do.

**Pull request closed without merging**